kratos/python/add_serializer_to_python.cpp

```diff
@@ -25,6 +25,39 @@
 
 namespace Kratos
 {
+
+        //this object is needed because data may contain binary data so the treatment of strings 
+    //in python makes impossible to use them (reasons related to locale)
```

Comment:
Looking at this twice, I think either this comment should go in the py::bytes cast or simply add this class in the pybind11 namespace (but not in the module) and return the py::byte array here